### PR TITLE
experimental background image command

### PR DESCRIPTION
**Pull request by indirectlylit**
### Summary

adds an experimental command to modify background image

### Reviewer guidance

* does this implementation make sense?
* does it work as expected?

----

### Contributor Checklist

- [x] Contributor has fully tested the PR manually
- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label

### Reviewer Checklist

- [ ] Automated test coverage is satisfactory
- [ ] Reviewer has fully tested the PR manually
- [ ] PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- [ ] External dependencies files were updated (`yarn` and `pip`)
- [ ] Documentation is updated
- [ ] Link to diff of internal dependency change is included
- [ ] CHANGELOG.rst is updated for high-level changes
- [ ] Contributor is in AUTHORS.rst

### CODE REVIEW

File: kolibri/content/management/commands/background.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+logger = logging.getLogger(__name__)
+
+class Command(BaseCommand):
+
+    def add_arguments(self, parser):
+        subparsers = parser.add_subparsers(dest='command', help="The following subcommands are available.")
+        movedir_subparser = subparsers.add_parser(
```
Comment (jamalex) [inline kolibri/content/management/commands/background.py]: Is this a copy-pasted name? Perhaps rename it to something more semantic here?

File: kolibri/content/management/commands/background.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        subparsers.add_parser(
+            name='reset',
+            cmd=self,
+            help="Set default"
+        )
+
+    def handle(self, *args, **options):
+        user_static_directory = os.path.join(settings.STATIC_ROOT, 'user_module')
+        if not os.path.exists(user_static_directory):
+            self.stderr.write(self.style.ERROR('\nStatic directory does not exist.'))
```
Comment (jamalex) [inline kolibri/content/management/commands/background.py]: Perhaps recommend in the error message that they start the server first?

Comment (jamalex) [reply kolibri/content/management/commands/background.py]: (so it gets created)

**Review by jamalex — APPROVED** [review]
Looks good! I suppose that'll get overwritten after an upgrade, or? We could probably find ways to avoid that, but that could wait until we decide to mainline this.

**Comment by rtibbles** [comment]
Buildkite failure is stochastic, restarting.

**Pull request merged**